minikube-nfs.sh

```diff
@@ -329,18 +326,15 @@ sudo mount -t nfs -o "$prop_mount_options" "$prop_nfshost_ip":\""$shared_folder"
 }
 
 # @info:    Restarts machine
-restartMachine()
+mountNFS()
 {
   if isPropertyNotSet $prop_profile_name; then
     echoError "'prop_profile_name' not set!"; exit 1;
   fi
 
-  echoInfo "Stopping machine ...\n"
-  minikube stop -p $prop_profile_name
-  echoSuccess "\t\t\t\t\t\t\tOK"
-
-  echoInfo "Starting machine ...\n"
-  minikube start -p $prop_profile_name
+  echoInfo "Mounting NFS ...\n"
+  minikube ssh -p $prop_profile_name \
+  "sudo /var/lib/boot2docker/bootlocal.sh"
```

Comment:
Let's move this one line to `configureBoot2Docker` and remove `restartMachine` altogether

Reply:
Updated.